tests/Unit/DataStructures/Test_OrientVariablesOnSlice.cpp

```diff
@@ -71,6 +71,7 @@ void check_orient_variables_on_slice(
   const auto& expected_coords = coords_on_neighbor_slice;
   for (size_t d = 0; d < SpatialDim; ++d) {
```

Comment:
Would `CHECK_ITERABLE_APPROX` work and also print out the desired info?

Reply:
It doesn't print out this info, just the values when the check fails.